app/models/game.rb

```diff
@@ -22,5 +23,45 @@ def coordinate_conflict?(test_coords)
     false
   end
 
+  def populate_game
+    # populate black pieces at top
+  
+    (0..7).each do |n|
+      pieces.create(type: "Pawn", color: "black", x_pos: n, y_pos: 6)
```

Comment:
The piece class shouldn't need a `type` attribute in the db. The point of setting up Single Table Inheritance is that each piece is a subclass that inherits from `Piece`. So instead of `pieces.create(...)` it should probably look something like `Pawn.create(color: 'black', x_pos: n, y_pos: 6, game: self)` or similar